test/OpenTelemetry.Instrumentation.EntityFrameworkCore.Tests/OpenTelemetry.Instrumentation.EntityFrameworkCore.Tests.csproj

```diff
@@ -1,9 +1,14 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
nit whitespace change

Reply:
Whis file, the encoding changed from UTF-8-BOM to UTF-8. 
Notepad++ shows this, as well as a hex editor.

I'll revert this change.